### PR TITLE
TT-5: UI components for sending broadcast messages

### PR DESCRIPTION
**Pull request by sghosalkar**
UI screenshots:
image

image

### CODE REVIEW

File: src/components/messages/multi-select-users.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const MultiSelectUsers = ({users, checkHandler, uncheckHandler}) => {
```
Comment (sghosalkar) [inline src/components/messages/multi-select-users.js]: @galen94 @tomirisk You should be able to reuse this component for your use cases of multi-selecting users for creating a chat group and for creating a private story.

Comment (galen94) [reply src/components/messages/multi-select-users.js]: Everything looks good. Thank you for creating the multi-select users component!

**Review by galen94 — APPROVED** [review]

**Comment by sghosalkar** [comment]
Added react-select for multi selecting users

image

File: src/components/messages/broadcast.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      });
+    })
+  }, []);
+
+  /**
+   * Sends the broadcast message
+   * @param message the message
+   * @param attachment the attachment if present
+   */
+  const send = (message, attachment) => {
```
Comment (sanatdeshpande1) [inline src/components/messages/broadcast.js:39]: Should we also have validation to check if the attachment is of any specific type? Or should we just allow files with any extensions?

**Review by sanatdeshpande1 — APPROVED** [review]
Looks good!

File: src/components/messages/broadcast.js (hunk 2 above)
Comment (sghosalkar) [reply src/components/messages/broadcast.js:39]: I don't think allowing a specific type of files will be of any benefit to us. The way we have built the components, we can keep this generic, more like a document sharing feature than just an image sharing feature

**Pull request merged**